src/database-controller/config/database-controller.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+service_type: "common"
+
+# general settings
+log-level: info
+recovery-mode: false
+k8s-connection-timeout-second: 120
```

Comment:
Not need manual recovery-mode if we already have auto recover in init container?

Reply:
btw Note that during recover do not serve any http request

---
In reply to: [466150363](https://github.com/microsoft/pai/pull/4761#discussion_r466150363) [](ancestors = 466150363)